module/zfs/range_tree.c

```diff
@@ -618,7 +618,7 @@ range_tree_verify_not_present(range_tree_t *rt, uint64_t off, uint64_t size)
 {
 	range_seg_t *rs = range_tree_find(rt, off, size);
 	if (rs != NULL)
-		panic("segment already in tree; rs=%p", (void *)rs);
+		panic("segment already in tree; rt=%px rs=%px", rt, (void *)rs);
```

Comment:
Isn't the `%px` a Linux-specific extension?

PS: Since this patch is already big, it would be nice to move all unrelated chunks (here and above) out.